docs/src/dev/io/graphson.asciidoc

```diff
@@ -2819,6 +2779,16 @@ The following `Bytecode` example represents the traversal of `g.V().hasLabel('pe
 }
 ----
 
+==== Merge
```

Comment:
Why no onCreate too?

Reply:
we only show one example for most of these. it's (hopefully) understood that any enum value of `Merge` would follow this pattern.